_adr/87/index.adoc

```diff
@@ -1,48 +1,452 @@
+* For *Broker* metrics, the metric prefix would be `kas_broker_`.
+* For *Topic* metrics, the metric prefix would be `kas_topic_`.
+
+### Converted Metrics
+At minimum, each metric should have an `instance_name` label that identifies the Kafka instance if possible.
+This would be useful if a user has multiple Kafka instances.
+If they want to integrate all of their Kafka metrics into one monitoring system, the `instance_name` can act as an identifier.
+
+#### Cluster Metrics
+|===
+|Old Metric Name | Revised Metric Name | Type | Old Labels | Revised Labels | Description
+
+|kafka_instance_max_message_size_limit
+|kas_cluster_message_size_limit_bytes
+|Gauge
+|Not available
+|instance_name
+|Maximum message size, in bytes, supported by the instance.
+
+|kafka_instance_partition_limit
+|kas_cluster_partition_limit
+|Gauge
+|Not available
+|instance_name
+|Maximum number of partitions per instance
+
+|kafka_namespace:haproxy_server_bytes_in_total:rate5m
+|kas_cluster_incoming_bytes_rate5m
+|Gauge
+|Not available
+|Not available (no label available to get the instance_name from)
+|Number of incoming bytes per second in the last five minutes per instance
+
+|kafka_namespace:haproxy_server_bytes_out_total:rate5m
+|kas_cluster_outgoing_bytes_rate5m
+|Gauge
+|Not available
+|Not available (no label available to get the instance_name from)
+|Number of outgoing bytes per second in the last five minutes per instance
+
+|kafka_topic:kafka_topic_partitions:count
+|kas_cluster_topic_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of topics, excluding system topics, per instance
+
+|kafka_topic:kafka_topic_partitions:sum
+|kas_cluster_topic_partition_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of topics across all topics, excluding system topics, per instance
+
+|consumergroup:kafka_consumergroup_members:count
+|kas_cluster_consumer_group_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of consumer groups, excluding system consumer groups, per instance
+
+|kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum
+|kas_cluster_connection_total
+|Gauge
+|Not available
+|instance_name
+|Total number of connections, excluding system connections apart from canary, per instance
+
+|kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum
+|kas_cluster_connection_creation_rate
+|Gauge
+|Not available
+|instance_name
+|Rate of connections established per second, excluding system connections apart from canary, per instance
+
+|kafka_instance_spec_brokers_desired_count
+|kas_cluster_desired_broker_total
+|Gauge
+|Not available
+|instance_name
+|Number of desired brokers per instance
+
+|===
+
+#### Broker Metrics
+|===
+|Old Metric Name | Revised Metric Name | Type | Old Labels | Revised Labels | Description
+
+|kubelet_volume_stats_available_bytes
```

Comment:
do we actually need this metric?.  we have kas_broker_storage_limit_bytes and kas_broker_storage_used_bytes - that is sufficient.

Reply:
I'm not too sure but it could be handy to not do any additional calculation to get this value? @riccardo-forina are you using this metric in the UI?

Reply:
No, but we point to that metric [in the guides](https://github.com/redhat-developer/app-services-guides/blob/2bfe9b92ab23f2002063581adae9b92bd65eede9/docs/kafka/metrics-monitoring-kafka/README.adoc#L128)

Reply:
thanks for confirming. I think we're ok to leave this in for now since it's already there, unless there's a strong need to remove it.

Reply:
Sounds good!